src/Features/Core/Portable/RemoveUnusedMembers/AbstractRemoveUnusedMembersDiagnosticAnalyzer.cs

```diff
@@ -207,7 +207,7 @@ private void AnalyzeFieldInitializer(OperationAnalysisContext operationContext)
                 {
                     foreach (var field in initializer.InitializedFields)
                     {
-                        OnSymbolUsage(field, ValueUsageInfo.Write);
+                        OnSymbolUsage(field, ValueUsageInfo.ValueWrite);
```

Comment:
this reads strangely to me.  the 'value' seems redundant since htis is already ValueUsageInfo.

Reply:
Sorry, I forgot to mention in the PR description - I intend to change the enum name to `SymbolUsageInfo` in a separate commit after the PR is reviewed - the enum is now not just for read/write information for values. Attempting to do so now caused lots of cascading changes in files, which would be a review hindrance.

Reply:
I will also rename `GetValueUsageInfo` to `GetSymbolUsageInfo`